_posts/2020-06-01-newsletter.md

```diff
@@ -0,0 +1,105 @@
+
+We released another major of Origami Build Tools. This release includes some significant maintenance work updating a number of outdated or deprecated dependencies. But more excitingly we've made improvements to the workflow for component maintainers too 🎉
+
+_If you contribute to Origami components or your team actively maintains one we recommend upgrading to v10. Please let us know if you have any issues or feedback._
+
+#### Simplified Interface
+
+To help engineers new to Origami components get up and running with component development we simplified the most common command. To start working on a component with Origami Build Tools it was common to run the following command:
+
+```
+obt demo --run-server --watch --verbose
+```
+
+That's a lot of options to understand at first. Instead we simplified this to:
+
+```
+obt develop
+```
+
+If you've never used Origami Build Tools before and would like to understand what this command does, [see the Origami Build Tools readme](https://github.com/Financial-Times/origami-build-tools#usage).
+
+#### Speed Improvements
+
+Origami Build Tools now compiles component code faster. It's an important metric for Origami Build Tools as it helps keep the feedback loop short between an engineer making a code change and seeing the result of that change.
+
+As an unscientific example on an FT MacBook Pro laptop: the quickest of three [o-table](https://github.com/Financial-Times/o-table) demo builds is 2.18 seconds over 4.11 seconds -- around a 53% improvement.
+
+We improved the performance of Origami Build Tools by creating two new packages:
+- [Financial-Times/scrumple](https://github.com/Financial-Times/scrumple#scrumple): a fast (and scrappy) JavaScript bundler for developing Origami components, based on a now-deleted tool called Pax.
+- [Financial-Times/sass](https://github.com/Financial-Times/sass): Prebuilt [dart-sass](https://github.com/sass/dart-sass) binaries available via NPM.
+
+## Special Thanks
+
+This month our special thanks goes to Rowan Manning! In case you missed the news, at the start of May Rowan moved from the Origami team (O&R) to Customer Products. So a big thank you to Rowan for all their work on and leading Origami over the years 😃 (And congratulations again to Jake Champion, who is our new Origami tech lead!)
+
+## Broader Update
+
+A digest of other things that have happened since our last update:
+
+- MAJOR: [Financial-Times/origami-bower-registry](https://github.com/Financial-Times/origami-bower-registry). Upgrade to node 12, drop support for previous versions.
+- NEW: [Financial-Times/origami-workshop](https://github.com/Financial-Times/origami-workshop): We created a small tool to help attendees of the Intro to Origami workshop complete [the manual build tutorial](https://origami.ft.com/docs/tutorials/manual-build/). The tool helps attendees focus on on building projects with Origami components, instead of setting up build tools which may differ depending on their team.
```

Comment:
```suggestion
- NEW: [Financial-Times/origami-workshop](https://github.com/Financial-Times/origami-workshop): We created a small tool to help attendees of the Intro to Origami workshop complete [the manual build tutorial](https://origami.ft.com/docs/tutorials/manual-build/). The tool helps attendees focus on building projects with Origami components, instead of setting up build tools which may differ depending on their team.
```